src/types/types.ts

```diff
@@ -32,3 +32,68 @@ export type StaticTokenInfo = {
 
 type KnownNetworkKeys = keyof typeof NETWORKS;
 export type KnownNetwork = typeof NETWORKS[KnownNetworkKeys];
+
+export type TotalPoolCommitments = [
+  ethers.BigNumber,
+  ethers.BigNumber,
+  ethers.BigNumber,
+  ethers.BigNumber,
+  ethers.BigNumber,
+  ethers.BigNumber,
+  ethers.BigNumber
+] & {
+  longMintSettlement: ethers.BigNumber;
+  longBurnPoolTokens: ethers.BigNumber;
+  shortMintSettlement: ethers.BigNumber;
+  shortBurnPoolTokens: ethers.BigNumber;
+  shortBurnLongMintPoolTokens: ethers.BigNumber;
+  longBurnShortMintPoolTokens: ethers.BigNumber;
+  updateIntervalId: ethers.BigNumber;
+}
```

Comment:
is this return from contracts?

Reply:
yeah I couldn't get the return type out of `PoolCommitter.totalPoolCommitments` and got sad to just did this :(